web/src/routes/backOffice/employees/list/EmployeesTable.svelte

```diff
@@ -0,0 +1,126 @@
+<script lang="ts">
+	import type {
+		Employee,
+		EmployeeSortableFields,
+	} from "../../../../domain/employees";
+	import Table from "../../../../lib/components/table/Table.svelte";
+	import type {
+		Columns,
+		SortingDirection,
+	} from "../../../../lib/components/table/types";
+	import { DEFAULT_PAGE_SIZE } from "../../../../lib/constants/pagination";
+	import { t } from "../../../../lib/utils/i8n";
+	import { rolesOptions } from "../constants/roles";
+	import employeesStore from "./employeesStore";
+	import { formatDate } from "../../../../lib/utils/date";
+	import { DateFormats } from "../../../../lib/constants/date";
+
+	const { loading, data, filters } = employeesStore;
+
+	const columns: Columns<Employee> = [
+		{
+			type: "accessor",
+			field: "firstName",
+			header: $t("employees.name"),
+			enableSorting: false,
+			enableFiltering: false,
+			cell(firstName, row) {
+				return `${firstName} ${row.lastName}`;
+			},
+		},
+		{
+			type: "accessor",
+			field: "username",
```

Comment:
Since the username field is the one that can be filtered, can we make it the first column in the table?

Reply:
@joaotomaspinheiro, I think that we should do the same thing for containers.
What do you think?